src/www/ui_components/app-root.js

```diff
@@ -298,49 +298,51 @@ export class AppRoot extends mixinBehaviors
         <poop-with-flies-dot-gif/>
       -->
 
-      <div id="logo-nav">
-        <img src\$="[[rootPath]]assets/logo-nav.png" alt="logo" id="logo">
+      <div style="height: 100%; overflow-y: auto;">
```

Comment:
You mind we add this into a css selector, rather than as a `style` property? they can be hard to override. Also does `overflow-y: scroll;` work?

Reply:
`scroll` does not look good because it will always show a scrollbar even when the space is enough.

For the css selector, we can do that, but wanna be sure about the benefits, because this `<div>` does not have any semantic meanings, it is just used for the layout purpose.

Reply:
True, `scroll` is a lot more obnoxious on windows.

You wouldn't be changing the `div` tag, just add a class to write the style against. It will remain non-semantic

Reply:
Yup, that's what I mean, since its only purpose is "layout", an inline style would make more sense to me to elaborate what this `<div>` is used for. If we use a css class, developers would need to navigate back and forth to understand why we have a `<div>` here.

Reply:
I just figured out another way, which is to name the css class as `.scrollable-drawer-root`, which can service both purposes (semantic in HTML and flexible in CSS).

Reply:
Updated the code.

Regarding the comment above, any `!important` in the code is a bad smell. If we do have a requirement to override the styles, we should put them into the CSS. What I was talking about before is, do we really need to override the layout styles? One of the use case I can think of is, using different layouts for different screen sizes.

Reply:
I think it's just one of those habit things. Junior devs (or in our case, other contributors) see inline styles and start copying that without thinking. Eventually it grows into a problem. We generally will need to override attributes, it's in the name (_Cascading_ Style Sheets) and it's kind of hubris to think we can predict which ones we'll have to override in the future. Who knows what features we'll be developing in 2022? Having a consistent approach to our CSS will serve us well.

I think the real real way to solve the problem that's irking you is to create a new web component primitive, specifically called `<scroll-viewer>`, but I'd argue that's overengineering, at this point

Reply:
😃 vote for `<scroll-viewer>`, but not in this pr.

Reply:
I believe the vision is to prefer to use material primitives where we can rather than rolling our own, so that's something to factor in as well